frontend/src/registration/actions.ts

```diff
@@ -176,7 +176,7 @@ export const submitRegistration = (
 
           dispatch({
             type: Types.REGISTRATION_VALIDATION_SUCCESS,
-            userData,
+            data: userData,
```

Comment:
We didn't face this issue before, was something updated here? Can you check if this is happening on prod?

Reply:
@giovannicimolin, this should be happening in prod as well, because the reducer is using `userData` and the ES6 automatic expansion currently passes the parameter as `userData: userData`.